number.go

```diff
@@ -273,6 +273,126 @@ func (n *Number) NotInDelta(value, delta float64) *Number {
 	return n
 }
 
+// InDeltaRelative succeeds if two numerals are within a relative delta of each other.
+//
+// The relative delta is expressed as a percentage (i.e. 0.01 is 1%).
+//
+// Example:
+//
+//	number := NewNumber(t, 123.0)
+//	number.InDeltaRelative(126.5, 0.03)
+func (n *Number) InDeltaRelative(value, delta float64) *Number {
+	opChain := n.chain.enter("InDeltaRelative()")
+	defer opChain.leave()
+
+	if opChain.failed() {
+		return n
+	}
+
+	if math.IsNaN(n.value) || math.IsNaN(value) || math.IsNaN(delta) {
+		opChain.fail(AssertionFailure{
+			Type:     AssertEqual,
+			Actual:   &AssertionValue{n.value},
+			Expected: &AssertionValue{value},
+			Delta:    &AssertionValue{relativeDelta(delta)},
+			Errors: []error{
+				errors.New("expected: numbers are comparable"),
+			},
```

Comment:
I suggest to refine error a bit, to something like:

```go
Errors: []error{
	errors.New("expected: can compare values with relative delta"),
	errors.New("actual value is NaN"), // included if n.value is NaN
	errors.New("expected value is NaN"), // included if value is NaN
	errors.New("delta is NaN"), // included if delta is NaN
},
```

(Feel free to reword if my English is poor).